tests/endtoend/test_linux_consumption.py

```diff
@@ -2,11 +2,11 @@
 # Licensed under the MIT License.
 import os
 import sys
-
 from unittest import TestCase, skipIf
+
```

Comment:
nit;Why this space?

Reply:
Pycharm added this. Probably since import are coming from different sources.